root/user/UserProfile.js

```diff
@@ -0,0 +1,520 @@
+              },
+            )}
+          </UserProfileProperty>
+
+          <UserProfileProperty name={l('Cancelled')}>
+            {exp.l(
+              '{count} ({view_url|view})',
+              {
+                count: formatCount($c, Number(editStats.cancelled_count)),
+                view_url: `/user/${encodeURIComponent(user.name)}/edits/cancelled`,
+              },
+            )}
+          </UserProfileProperty>
+
+          <UserProfileProperty name={l('Last 24 hours')}>
+            {exp.l(
+              '{count} ({view_url|view})',
+              {
+                count: formatCount($c, Number(editStats.last_day_count)),
+                view_url: `/search/edits?conditions.1.operator=%3E&conditions.1.field=open_time&conditions.1.args.0=24+hours+ago&combinator=and&conditions.0.name=${encodeURIComponent(user.name)}&conditions.0.field=editor&order=desc&conditions.0.args.0=${encodeURIComponent(user.name)}&conditions.0.operator=%3D&negation=0&auto_edit_filter=`,
+              },
+            )}
+          </UserProfileProperty>
+        </tbody>
+      </table>
+
+      <table
+        className="statistics"
+        title={l('This table shows a summary of votes cast by this editor.')}
+      >
+        <thead>
+          <tr>
+            <th colSpan="3">
+              {exp.l(
+                'Votes ({view_url|view})',
+                {
+                  view_url: `/user/${encodeURIComponent(user.name)}/votes`,
+                },
+              )}
+            </th>
+          </tr>
+          <tr>
+            <th id="table_vote_summary_vote" />
+            <th id="table_vote_summary_recent">
+              {l('Last 28 days')}
+            </th>
+            <th id="table_vote_summary_overall">
+              {l('Overall')}
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {votes.map(voteStat => (
+            <tr key={voteStat.name}>
+              <th headers="table_vote_summary_vote">
+                {voteStat.name}
+              </th>
+              <td headers="table_vote_summary_recent">
+                {formatCount($c, Number(voteStat.recent.count))}
+                {' '}
+                {bracketedText(texp.l('{percentage}%', {percentage: voteStat.recent.percentage}))}
+              </td>
+              <td headers="table_vote_summary_overall">
+                {formatCount($c, Number(voteStat.all.count))}
+                {' '}
+                {bracketedText(texp.l('{percentage}%', {percentage: voteStat.all.percentage}))}
+              </td>
+            </tr>
+          ))}
+        </tbody>
+        <tfoot>
+          <tr>
+            <th headers="table_vote_summary_vote">
+              {voteTotals.name}
+            </th>
+            <th className="totals" headers="table_vote_summary_recent">
+              {formatCount($c, Number(voteTotals.recent.count))}
+              {' '}
+              {bracketedText(l('100%'))}
```

Comment:
I agree we should use `formatPercentage` here as wondered in https://github.com/metabrainz/musicbrainz-server/pull/1089#discussion_r290687724